doc/en/components/grids/_shared/remote-data-operations.md

```diff
@@ -8,7 +8,7 @@ namespace: Infragistics.Controls
 
 # {Platform} {ComponentTitle} Remote Data Operations
 <!-- Angular -->
-The {Platform} {ComponentName} Remote Data {ProductName} supports operations such as remote virtualization, remote sorting, remote filtering and others. This allows the developer to perform these tasks on a server, retrieve the data that is produced and display it in the `{ComponentName}`.
+The {ProductName} Remote Data Operations feature in {Platform} {ComponentTitle} supports operations such as remote virtualization, remote sorting, remote filtering and others. This allows the developer to perform these tasks on a server, retrieve the data that is produced and display it in the `{ComponentTitle}`.
```

Comment:
Whats the purpose of `{ComponentTitle}`? Is it to create a link to the API docs? If so it wont work for hierarchical and tree grids because ComponentTitle maps to `Hierarchical Grid` and `Tree Grid` and not `IgrTreeGrid`